src/rekapi.actor.js

```diff
@@ -894,6 +895,28 @@ rekapiModules.push(function (context) {
   };
 
   /*!
+   * Set the actor to be active or inactive stating at `millisecond`.
+   * @method setActive
+   * @param {boolean} isActive Whether the actor should be active or inactive
+   * @param {number} millisecond The time at which to change the actor's active state
+   * @chainable
+   */
+  Actor.prototype.setActive = function (isActive, millisecond) {
+    var activeProperty = this._propertyTracks._active
+        && this.getKeyframeProperty('_active', millisecond);
+
+    if (activeProperty) {
+      activeProperty.value = isActive;
+    } else {
+      activeProperty = new Rekapi.KeyframeProperty(
+        millisecond, '_active', isActive, DEFAULT_EASING);
```

Comment:
If I'm understanding this correctly, this `KeyframeProperty` won't be used to render anything, so you can omit the `DEFAULT_EASING` parameter since the `KeyframeProperty` will handle that anyways.